package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "zipcodeapi",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "main": "index.js",
   "repository": "ssh://git@github.com/golmansax/zipcodeapi.git",
```

Comment:
@walliby Can you also change this URL to be `/boostmyschool/zipcodeapi`

Reply:
yep. https://github.com/boostmyschool/zipcodeapi/pull/1/commits/2ac9838a7c7faf17663ab107cd8817222bb9982e